### PR TITLE
Updated docs for release v0.1.34

### PR DESCRIPTION
**Pull request by AverageMarcus**
## Change description
Updates the docs for the new release version

/hold for new release being available

### CODE REVIEW

File: docs/book/src/capi/releasing.md [hunk 1]
```diff
@@ -77,14 +77,14 @@ Wait for this PR to merge before communicating the release to users, so image-bu
...
+Image-builder v0.1.35 is now available: https://github.com/kubernetes-sigs/image-builder/releases/tag/v0.1.35
 Thanks to all contributors!
 ```
 
-[v0.1.33]: https://github.com/kubernetes-sigs/image-builder/releases/tag/v0.1.33
+[v0.1.34]: https://github.com/kubernetes-sigs/image-builder/releases/tag/v0.1.34
 [#image-builder channel]: https://kubernetes.slack.com/archives/C01E0Q35A8J
 [Personal access tokens]: https://github.com/settings/tokens
-[post-image-builder-push-images]: https://testgrid.k8s.io/sig-cluster-lifecycle-image-pushes#post-image-builder-push-images
+[post-image-builder-push-images]: https://prow.k8s.io/?repo=kubernetes-sigs%2Fimage-builder&type=postsubmit&job=post-image-builder-push-images
```
Comment (mboersma) [inline docs/book/src/capi/releasing.md:87]: 👍🏻

**Review by mboersma — APPROVED** [review]
/lgtm
/approve

File: docs/book/src/capi/releasing.md (hunk 1 above)
Comment (AverageMarcus) [reply docs/book/src/capi/releasing.md:87]: Yeah, I asked about it in SIG-Testing and learnt more about how Prow and Testgrid fit together!

**Pull request merged**